src/main/java/com/znsio/teswiz/runner/DeviceSetup.java

```diff
@@ -261,4 +271,26 @@ static void cleanupCloudExecution() {
                         String.format("Provided cloudName: '%s' is not supported", cloudName));
         }
     }
+
+    static ArrayList<String> setupIOSExecution() throws IOException {
```

Comment:
why are we throwing exceptions? Throw a custom runtime exception

Reply:
Not required here, handled in the calling method verifyAppExistsAtMentionedPath();